server/graphql/v2/interface/Account.js

```diff
@@ -169,8 +169,16 @@ const accountFieldsDefinition = () => ({
   },
   transferwise: {
     type: TransferWise,
-    resolve(collective) {
-      return collective;
+    async resolve(collective) {
+      const connectedAccount = await models.ConnectedAccount.findOne({
+        where: { service: 'transferwise', CollectiveId: collective.id, deletedAt: null },
+      });
+      if (connectedAccount) {
+        collective.connectedAccount = connectedAccount;
```

Comment:
This is a bit hacky, we should avoid adding fields on models that are not defined. What will happen if a different part of the code starts using the same field for storing the Twitter account because it needs it? Without the context, how will the person reading the code for `transferwise/index.ts` understand where the field is defined?
Also it will probably trigger an error if we move the file to typescript. 

What was wrong with loading the proper `connectedAccount` in `transferwise/index.ts` like you did before?

Reply:
Agreed, this implementation is hacky, I'll just use Sequelize's include to preload connected accounts and use `find()` in `transferwise/index.ts`.
The idea here is validating the collective is actually connected to TransferWise before calling related methods so we can cover the case where the collective is not using TW.
It is also an optimization so we can load it once and use it later.